src/mender-update/CMakeLists.txt

```diff
@@ -56,7 +56,7 @@ target_sources(update_module PRIVATE
 target_compile_options(update_module PRIVATE ${PLATFORM_SPECIFIC_COMPILE_OPTIONS})
 
 add_library(mender_http_resumer STATIC
-  http_resumer/http_resumer.cpp
+  ../common/http_resumer/http_resumer.cpp
 )
 target_link_libraries(mender_http_resumer PUBLIC
   common_http
```

Comment:
This library definition now needs to be moved to `src/common/CMakeLists.txt`.

Generally speaking we build CMake top-down approach, which means that child directories should not require files from parent directories.